.github/workflows/ci.yml

```
@@ -1,50 +1,41 @@
       - name: Run tests
         run: pytest
-
       - name: Run lint
         run: |
           ruff check .
           mypy app
-
-      - name: Qodo Review
-        if: github.event_name == 'pull_request'
+      - name: Install Node.js
+        if: ${{ github.event_name == 'pull_request' && secrets.QODO_API_KEY != '' }}
```

actionlint (1.7.7)

context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)